docs/kubectl-port-forward.md

```diff
@@ -0,0 +1,65 @@
+## kubectl port-forward
+
+Forward 1 or more local ports to a container.
+
+### Synopsis
+
+Forward 1 or more local ports to a container.
+Examples:
+  $ kubectl port-forward -p mypod -c mycontainer 5000 6000
```

Comment:
I was just following the method for command execution, but now that you mention it, since the containers all share a single network namespace, I should go back and remove the container parameter for port forwarding.